content_schemas/formats/email_alert_signup.jsonnet

```diff
@@ -37,7 +37,6 @@
         email_alert_type: {
           type: "string",
           enum: [
-            "topics",
```

Comment:
Let's cherry pick https://github.com/alphagov/publishing-api/pull/2669/commits/ad572f9b2142c2d0377ec4026a366c4c9b9b8545 and  https://github.com/alphagov/publishing-api/pull/2669/commits/2a8893b1b39a60c845542f76ed94b4fc9b3d5cfd into its own work: "Remove `topics` from email_alert_signup schema and associated examples". I'm not sure we've removed this field from the publishing applications as per the documentation for [removing-a-field-from-a-content-schema ](https://github.com/alphagov/publishing-api/blob/main/docs/content_schemas/changing-an-existing-content-schema.md#removing-a-field-from-a-content-schema) so it would be nice to have this as a standalone PR, so we can link to relevant changes in other apps.